File: hash_tables/hash_table.py
# Creating a hash table using dictionaries (objects)
hash_table = {}

# Adding values to the hash table
hash_table['key1'] = 'value1'
hash_table['key2'] = 'value2'
hash_table['key3'] = 'value3'

# Retrieving values from the hash table
print(hash_table['key1'])  # Output: value1
print(hash_table['key2'])  # Output: value2

# Deleting a key from the hash table
del hash_table['key3']

# Checking if a key exists
print('key3' in hash_table)  # Output: False

File: hash_tables/hash_table.js
// Creating a hash table using objects
const hashTable = {};

// Adding values to the hash table
hashTable['key1'] = 'value1';
hashTable['key2'] = 'value2';
hashTable['key3'] = 'value3';

// Retrieving values from the hash table
console.log(hashTable['key1']);  // Output: value1
console.log(hashTable['key2']);  // Output: value2

// Deleting a key from the hash table
delete hashTable['key3'];

// Checking if a key exists
console.log('key3' in hashTable);  // Output: false
